docs/SETUP.md

```diff
@@ -10,7 +10,7 @@ Link-SF is designed to be free to setup and free to run. Since hosting files is
 
 * [Firebase](https://firebase.google.com/console):  Firebase is the only required account, and it's completely free.  It will provide everything we need to get up and running, including database storage, authentication, and hosting.
 
-  **Note**  Every administrator on your account will need to be created under the Authentication panel in your new Firebase Console.  You won't be able to adjust your production data from the app without it!
+  **Note**  Every administrator on your account will need to be created under the Authentication panel in your new Firebase Console.  You won't be able to adjust your production data from the app without it!  For information on how to set up users, see the [Management documentation](https://github.com/zendesk/linksf/blob/master/docs/MANAGE.md) under *Set up a Firebase user*.
```

Comment:
can link directly to the section with 

```
linksf/blob/master/docs/MANAGE.md#set-up-a-firebase-user
```

notice its also a relative link so you don't need the full https foobar